dear_petition/petition/models.py

```diff
@@ -123,8 +123,6 @@ class OffenseRecord(models.Model):
     action = models.CharField(max_length=256)
     severity = models.CharField(max_length=256)
     description = models.CharField(max_length=256)
-    plea = models.CharField(max_length=256, blank=True)
-    verdict = models.CharField(max_length=256, blank=True)
```

Comment:
Was curious why this was being removed. Were these fields ever used? Did #163 remove the need for these fields?

Reply:
I removed these fields here because they were redundant with the plea and verdict on the Offense model. My current understanding is that plea and verdict is an offense-level field, so I removed them here but they are still on the Offense model.